helm/values.yaml

```diff
@@ -6,13 +6,17 @@ psql:
   job:
     backoffLimit: "0"
     ttlSecondsAfterFinished: "10"
-    activeDeadlineSeconds: "1200"
+    activeDeadlineSeconds: "1800"
     container:
       image:
         repository: "docker-folio.dev.folio.org/org/folio/psql-dump-helm"
         pullPolicy: Always
-        tag: ""
-      envVarsSecret: "s3-postgres-backups-credentials"
+        tag: "1.0.0-RANCHER-313-20220628124331"
```

Comment:
I guess tag be a variable?

Reply:
No.
We will set some latest stable version of chart, that will dynamically provide to tag value of chart that will be equal to value of appropriate image in registry. 
spec:
      containers:
        - name: {{ .Chart.Name }}
          image: "{{ .Values.psql.job.container.image.repository }}:{{ .Values.psql.job.container.image.tag | default .Chart.Version }}"